.travis.yml

```
@@ -97,6 +97,17 @@ jobs:
       script:
           - make $MAKEJOBS
           - make $MAKEJOBS bench
+    - stage: JEMALLOC
+      os: linux
+      compiler: gcc
+      before_install:
+          - sudo apt-get update
+          - sudo apt-get install -y libjemalloc-dev
+      before_script:
+          - export COVERAGE=OFF
+      script:
+          - make $MAKEJOBS LDLIBS=-ljemalloc
+          - make check $MAKEJOBS VERBOSE=1
```

```suggestion
          - make check $MAKEJOBS  LDLIBS=-ljemalloc VERBOSE=1
```
And remove the line 109.